### PR TITLE
Added a CheatSheet for states

### PR DESCRIPTION
**Pull request by Midhun-Jo-Antony**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Capitals Of Indian States Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

It adds a cheatsheet about indian capitals

###### Which issues (if any) does this fix?

---
https://duck.co/ia/view/capital_states_india_cheatsheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@Midhun-Jo-Antony Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/capital_of_indian_state.json [hunk 1]
```diff
@@ -0,0 +1,125 @@
+{
+    "id": "Capital_States_India_Cheatsheet",
+    "name": "Capital of Indian States",
+    "metadata": {
+        "sourceName": "Capitals",
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/capital_of_indian_state.json]: Source name here should be `Wikipedia`

**Comment by gaulrobe** [comment]
@Midhun-Jo-Antony a better name for this file might be `indian_state_captials.json` so that the main trigger here is `indian state capitals cheat sheet`

**Comment by moollaza** [comment]
@Midhun-Jo-Antony thanks a lot for contributing! Unfortunately though, we already have a cheat sheet for this:

https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/json/states-of-india.json

If there's anything missing from there please add it. For now I'm closing this as it's a duplicate.

**Comment by moollaza** [comment]
My mistake, I was a little too quick, it looks like the sources are the same, but the content is different here. Please, carry on!

**Comment by gaulrobe** [comment]
@moollaza I think you were right... the other cheat sheet is `states of india` but it shows state and captials. Maybe we should add an alias to the other one to support `capitals of india`-esque queries?

**Comment by jophab** [comment]
@moollaza @Midhun-Jo-Antony  The contents are same in both. So it is better to add new ones to old cheat sheet

**Comment by Midhun-Jo-Antony** [comment]
@moollaza @gaulrobe i looked duckduckgo and there is already a similar cheat sheet for this.....states of india cheat sheet.

**Comment by zekiel** [comment]
@Midhun-Jo-Antony  what can we help you with? Maybe something with the [programming mission](https://duck.co/blog/post/297/help-for-programmers)?

**Pull request closed without merging**